src/Controller/ResourcesController.php

```diff
@@ -50,25 +36,19 @@ public function resourcesAction(Request $request) : Response
             ArticleSection::basic('Images', 2, $this->render(
                 new Paragraph('A picture is worth a thousand words and we use imagery to create impact and capture the imagination of our readers. In our Flickr channel you’ll find images carefully selected from research articles or specifically created, available to use under the CC BY 2.0 licensing agreement.'),
                 Listing::unordered([
-                    '<a href="https://www.flickr.com/photos/128643624@N07/">Visit Flickr channel</a>',
+                    '<a href="https://www.flickr.com/photos/109374423@N04/albums">Visit Flickr channel</a>',
                 ], 'bullet')
             )),
             ArticleSection::basic('The eLife logo', 2, $this->render(
-                new Paragraph('The eLife logo is available to download below. Please refer to the brand guide for usage guidelines and ensure you are downloading the correct file type for your purpose.'),
+                new Paragraph('The eLife logo is available to download below. Please ensure you are downloading the correct file type for your purpose.'),
                 ArticleSection::basic('Full-colour horizontal', 3, $this->render(
                     Listing::unordered([
-                        '<a href="'.$this->generateDownloadLink('https://cdn.elifesciences.org/style-guide-images/elife-full-color-horizontal.eps').'">Download .eps for print</a>',
-                        '<a href="'.$this->generateDownloadLink('https://cdn.elifesciences.org/style-guide-images/elife-full-color-horizontal.jpg').'">Download .jpg for web</a>',
-                        '<a href="'.$this->generateDownloadLink('https://cdn.elifesciences.org/style-guide-images/elife-full-color-horizontal.png').'">Download .png with transparent background</a>',
+                        '<a href="'.$this->generateDownloadLink('https://cdn.elifesciences.org/style-guide-images/elife-full-color-horizontal-2020.eps').'">Download .eps for print</a>',
+                        '<a href="'.$this->generateDownloadLink('https://cdn.elifesciences.org/style-guide-images/elife-full-color-horizontal-2020.jpg').'">Download .jpg for web</a>',
```

Comment:
Needs to match https://github.com/elifesciences/issues/issues/5484#issuecomment-595684138

Reply:
We cannot match these as they are in then CDN and it will change them straight away and they are not going live until Monday.

Reply:
The comment is about adding an SVG version.

Reply:
Sorry. misread